### PR TITLE
ON-4988 # Added tenant support, updated flow, added tests

### PR DESCRIPTION
**Pull request by Zaxist**

### CODE REVIEW

File: lib/scope.js [hunk 1]
```diff
@@ -5,8 +5,9 @@ const objectMerge = require('object-merge')
 
 const configHelper = require('./utils/config-helper')
 
-const CLIENT_CLI_SERVICE_DOMAIN = 'auth-api.blinkm.io'
-const CLIENT_CLI_SERVICE_ORIGIN = `https://${CLIENT_CLI_SERVICE_DOMAIN}`
+// const CLIENT_CLI_SERVICE_DOMAIN = 'auth-api.blinkm.io'
+// const CLIENT_CLI_SERVICE_ORIGIN = `https://${CLIENT_CLI_SERVICE_DOMAIN}`
```
Comment (mymattcarroll) [inline lib/scope.js]: Remove commented out code

**Comment by mymattcarroll** [comment]
There are flow errors, I would suggest reverting back to the flow version we were using before.

**Comment by Zaxist** [comment]
> There are flow errors, I would suggest reverting back to the flow version we were using before.

Ah, of course

**Comment by Zaxist** [comment]
Not sure what this is about. All of the tests failing failed on master too.

**Pull request merged**